src/contributing.md

```diff
@@ -1,5 +1,5 @@
 # Contributing to Alan
 
-Like most other open source projects, contributing to the Alan compiler or runtime requires a [CLA Aggreement](https://cla-assistant.io/alantech/) and here is a current [list of good starter tasks](https://github.com/alantech/alan/contribute).
+Like most other open source projects, contributing to the Alan compiler or VM requires a [CLA Aggreement](https://cla-assistant.io/alantech/) and here is a current [list of good starter tasks](https://github.com/alantech/alan/contribute).
```

Comment:
Nit: AVM

Reply:
Alan AVM seems redundant even if "compiler or" is in between

Reply:
why not drop `Alan` then? It's obvious what compiler we're talking about here.

Reply:
sure.

Reply:
Plain "VM" has colloquially come to mean "a virtual machine to emulate a whole computer", not "a program that executes some sort of binary instruction set", so plain "VM" could itself be confusing here in a way AVM is not.

Reply:
Ah I see your point